internal/provider/signin_policy_resource.go

```diff
@@ -44,17 +45,16 @@ func (r *SignInPolicyResource) Schema(_ context.Context, _ resource.SchemaReques
 			// ID field is required for Terraform Framework acceptance testing.
 			"id": schema.StringAttribute{
 				Description: "This identifier is generated by Terraform provider",
-				Optional:    true,
 				Computed:    true,
 				// this is a static identifier to identify the SignInPolicy resource in the state,
-				Default: stringdefault.StaticString("signinpolicy_identifier"),
+				Default: stringdefault.StaticString(getTenantId() + ":signin_policy"),
```

Comment:
Hmm - I wasn't imagining this working with multiple providers - my assumption was that the Default would have to be removed and we would just set the value. I think I'd still lean towards that to be safe (meaning please just remove this line).

Reply:
Do you mean different versions of Terraform as multiple providers?  If it is, currently it is only running for the 1.9 version in CI
https://github.com/Aembit/terraform-provider-aembit/blob/94b233a55b7242ccf1b80fc52675d1926d515a2e/internal/provider/signin_policy_resource_test.go#L16

And when I remove this, I get this error locally

`Terraform v1.9.5
    signin_policy_resource_test.go:23: Step 1/3 error: Error running apply: exit status 1

        Error: Provider returned invalid result object after apply

        After the apply operation, the provider still indicated an unknown value for
        aembit_signin_policy.test.id. All values must be known after apply, so this
        is always a bug in the provider and should be reported in the provider's own
        repository. Terraform will still save the other known object values in the
        state.
--- FAIL: TestAccSigninPolicy (2.76s)`

So there should be a default value for ID